framework/src/play/mvc/Scope.java

```diff
@@ -37,6 +34,19 @@
     public static final boolean SESSION_SEND_ONLY_IF_CHANGED = Play.configuration
             .getProperty("application.session.sendOnlyIfChanged", "false").toLowerCase().equals("true");
 
+    public static SessionStore sessionStore = createSessionStore();
+
+    private static SessionStore createSessionStore() {
+        String sessionStoreParameter = Play.configuration.getProperty("application.session.store", "cookie");
```

Comment:
I thought the simpler parameter is more consistent with other parameters, e.g. memcached